_episodes_rmd/07-genomic-ranges.Rmd

```diff
@@ -336,7 +336,7 @@ overlaps_iranges <- findOverlaps(query = query_iranges, subject = demo_iranges)
 overlaps_iranges
 ```
 
-The results are returned in the form of a `Hits` object, which we has not
+The results are returned in the form of a `Hits` object, which we have not
```

Comment:
I don't think this should be plural here. The way I wrote/read this is "a Hits object, an S4 class that has not been introduced yet."

What do you think? If so, Can I let you adjust this one more time?

Reply:
Yes but the way it was originally written was in the active form, because of the "we".
So either the "we" should be removed, or the "has" should be replaced by "have", because "we has" does not seem correct to me :)

Reply:
Oh wait, I'm blind, I didn't see the "we". You're right. All good!